remote-query/remote-query-client/src/main/resources/org/infinispan/query/remote/client/query.proto

```diff
@@ -88,14 +88,13 @@ message FilterResult {
 }
 
 /**
- * @TypeId(1000104)
+ * @TypeId(4403)
  */
 message ContinuousQueryResult {
 
-   /* @TypeId(1000105) */
```

Comment:
This enum is never encountered as a top level value in a message, so no need for it to have a TypeId